packages/nexus-sdk/src/View/__tests__/view.spec.ts

```diff
@@ -28,6 +31,16 @@ describe('Views', () => {
       );
       expect(fetchMock.mock.calls[0][1].method).toEqual('GET');
     });
+
+    it('should make httpGet call with the right header', async () => {
+      fetchMock.mockResponseOnce(JSON.stringify({ data: '' }));
```

Comment:
I think we should only check for header here(like I have mentioned in comments for archive). Other parts are tested in different test cases.